Dockerfile_amd

```diff
@@ -318,10 +318,11 @@ COPY --from=moe-kernels /usr/src/moe-kernels/build/lib.linux-x86_64-cpython-311
 COPY proto proto
 COPY server server
 COPY server/Makefile server/Makefile
+ENV UV_SYSTEM_PYTHON=1
 RUN cd server && \
     make gen-server && \
-    pip install -r requirements_rocm.txt && \
-    pip install ".[accelerate, compressed-tensors, peft, outlines]" --no-cache-dir
+    pip install -U pip uv && \
```

Comment:
```suggestion
```

Reply:
No. I know `uv` is already installed by `make gen-server`. But this is not important as it's totally opaque from here.

Leaving it here keeps things explicit for the reader that it gets downloaded to be used.